src/js/components/Video/StyledVideo.js

```diff
@@ -35,6 +35,14 @@ const StyledVideoContainer = styled.div`
   overflow: hidden;
   position: relative;
   ${genericStyles};
+  &:focus {
```

Comment:
I like the idea of the line for the focus just thinking is someone didn't want this or wanted a horizontal line